lib/writeGltf.js

```diff
@@ -19,31 +28,24 @@ function writeGltf(gltf, outputPath, embed, embedImage, createDirectory, callbac
     if (outputExtension !== '.gltf') {
         throw new DeveloperError('Invalid output path extension.');
     }
+
     // Create the output directory if specified
     if (createDirectory) {
         outputPath = path.join(path.dirname(outputPath), 'output', path.basename(outputPath));
+        fsExtra.ensureDirSync(path.dirname(outputPath));

```

Comment:
I had to do this due to `writeSource` using `fs.writeFile` (not fsExtras), but a better solution is probably just to use fsExtras in `writeSource`, so I'll go ahead and do that.